### PR TITLE
Enable config file to parse environment variables.

### PR DESCRIPTION
**Pull request by jnegara**
Environment variables can now be used in the config file such that `%(ENV_VAR)s` will be parsed as the value of the environment variable `ENV_VAR`.

### CODE REVIEW

File: tabpy/tabpy_server/app/app.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def _parse_config(self, config_file):
         self.python_service = None
         self.credentials = {}
 
-        parser = configparser.ConfigParser()
+        parser = configparser.ConfigParser(os.environ)
```
Comment (0golovatyi) [inline tabpy/tabpy_server/app/app.py]: In `set_parameter` we specifically check for environment variables with the same names and config file parameters. Do we remove that check now?

Comment (jnegara) [reply tabpy/tabpy_server/app/app.py]: It looks to me like we only look for environment variables if the parameter is not set in the config.

https://github.com/tableau/TabPy/blob/dev/tabpy/tabpy_server/app/app.py#L179

Comment (jnegara) [reply tabpy/tabpy_server/app/app.py]: Misunderstood your comment - as a side effect of my change we no longer need to check for environment variables.

Removed the additional check.

**Comment by jnegara** [comment]
For the record, I am seeing this same test failure in the dev branch.
https://travis-ci.com/tableau/TabPy/builds/124143059

**Pull request merged**